index.js

```diff
@@ -124,28 +124,33 @@ function keydown(event: KeyboardEvent) {
   // Ignore key presses from nested details.
   if (details.querySelector('details[open]')) return
 
+  let target
   switch (event.key) {
     case 'Escape':
       close(details)
       event.preventDefault()
       break
     case 'ArrowDown':
-      sibling(details, true).focus()
+      target = sibling(details, true)
+      if (target) target.focus()
```

Comment:
Creating a block inside the `case` branches will let us reduce the scope of `target`.

```js
switch (event.key) {
  case 'ArrowDown': { // <--- block scope here
    const target = sibling(details, true)
  }
  break
}
```